tagging_tools/dump_tags.cc

```diff
@@ -0,0 +1,127 @@
+/* Copyright © 2017-2019 The Charles Stark Draper Laboratory, Inc. and/or Dover Microsystems, Inc. */
```

Comment:
There is an existing tool, md_firmware_test, which prints tags in a similar way for taginfo files generated with the -s/--soc flag. Could you consolidate these two tools? (I think dump_tags is a better name)

Reply:
They could be.  What would be the best way to determine which format the tag file is in (simulation vs soc)?  Command line option, header flag, or something else?

Reply:
Command line option will work fine.

Reply:
Added a firmware option to dump_tags and replaced md_firmware_test with a wrapper script for backwards compatibility.